bower.json

```diff
@@ -35,6 +35,7 @@
 
         "bootstrap": "3.3.7",
         "jstree": "~3.3.3",
+        "jstree-actions": "orangejenny/jstree-actions#make-module",
```

Comment:
This can be set back to the original repo. May be worth asking the guy to make a tag first though